test/dir-list.test.js

```diff
@@ -471,7 +471,7 @@ t.test('dir list json format - extended info', t => {
           t.equal(response.statusCode, 200)
           const bodyObject = JSON.parse(body.toString())
           t.equal(bodyObject.dirs[0].name, 'empty')
-          t.equal(typeof bodyObject.dirs[0].stats.atime, 'string')
+          t.equal(typeof bodyObject.dirs[0].stats.atimeMs, 'number')
```

Comment:
I believe there is bug on `Node.js` side, `JSON.stringify` do not print `atime` directly.